### DISCUSSION TITLE
Streets of Rage Remake Displaying "Gamepad not detected" but controls work fine

### DISCUSSION THREAD

**Discussion by RocketsMeowth** [eduardofilo/RG350_adam_image#108] [Ports]
Hello Eduardo, 

I am loving the adam image, but when I play streets of rage remake it always displays "gamepad not detected" even though the controls work fine. 

There aren't any options in the game control settings I can find to make it recognize the gamepad even though it seems it already does. 

Do you have any idea how to force this opk to not display the message?

Thanks for everything!
Meowth

**Comment by eduardofilo** [comment]
Hi. I have managed to make the warning disappear. What I have done is the following:

1. Go to `Options > Controls` and invoke `Default controls`.
2. The control mapping converts to this:
    ![screenshot001](https://user-images.githubusercontent.com/2923728/140615977-abfd32f8-8a4b-4b83-9118-393ccc4ca334.png)
3. Press Start to show the `Keyboard/Gamepad` selector. Press Start again to select `Keyboard`. That enable the key mapper.
4. Map all the keys. To change a particular key, press Start and then the key to be mapped.
5. When all keys are mapped, use `Exit` to exit.

Now the warning `Gamepad 1 not found` doesn't show up. Although I have not found how the keys correspond to the original mapping. I have a hard time figuring it out because I'm not used to the game.

To start over if things get messy, you only need to delete the `/media/data/local/home/.sorrv5/` directory.

I must say that this game makes that the screen of my RG280V gets crazy. I have to force the shutdown of the system (holding Power) to recover.
